src/conversion_helper.js

```diff
@@ -367,6 +367,19 @@ const unitLookupList = [
       }
     },
     "ignoredUnits" : [/° ?C/, "degrees? c", "celsius", "kelvin"]
+  },
+  {
+    "imperialUnits" : [/ac/, /acres?/],
+    "standardInputUnit" : " acres",
+    "isInvalidInput" : isZeroOrNegative,
+    "isWeaklyInvalidInput" : isHyperbole,
+    "conversionFunction" : (i) => createMap(i * 0.00404685642, " km^2"),
+    "ignoredUnits" : [
+      /[sS]quare [kK]ilometers?/,
```

Comment:
You don't need to specify case in these regular expressions because it does a case-insensitive match [later on](https://github.com/annuhdo/metric_units_reddit_bot/blob/a1c21ba6790048cc17ebcfa6f3a1bcba6e49b88e/src/conversion_helper.js#L431). 

But this does bring up a great point that given the way this code is currently architected, it is pretty unclear how it behaves (since the match happens so far away). As a stop-gap, can you add some documentation [here](https://github.com/annuhdo/metric_units_reddit_bot/blob/a1c21ba6790048cc17ebcfa6f3a1bcba6e49b88e/src/conversion_helper.js#L75) saying it doesn't need to be case sensitive? 

I'm a fan of code standing alone without documentation though, so I added a story [here](https://www.pivotaltracker.com/story/show/151586691) to try to figure out a solution to make the code more clear... perhaps naming it `ignoredUnits-caseInsensitiveRegexArray` or something? Is that too verbose? Haha, something to think about...

Reply:
🤔  I like the name change, since based on that name I would know that it's case insensitive and it's a regex array.